fixbackend/cloud_accounts/service_impl.py

```diff
@@ -275,6 +286,11 @@ async def send_pub_sub_message(
                             next_scan=event.next_run,
                         ),
                     )
+                if first_collect:
+                    text = "Your first security scan is finished! You can now view the results in the FIX console."
```

Comment:
let's not put human readable emails in code - we should use message or language files.